music21/features/jSymbolic.py

```diff
@@ -607,30 +591,45 @@ def process(self):
 
 class DurationOfMelodicArcsFeature(featuresModule.FeatureExtractor):
     '''
-    Average number of notes that separate melodic peaks and troughs in any channel.
+    Average number of notes that separate melodic peaks and troughs
+    in any part. This is calculated as the total number of intervals
+    (not counting unisons) divided by the number of times the melody
+    changes direction.
+    
+    >>> # C D E D C E D C C
```

Comment:
Comments are better done without ">>> #"  There's no reason to put them with the code inside a docstring unless they belong on the same line as code